src/docs/user/references/ApplicationConnectors.adoc

```diff
@@ -0,0 +1,80 @@
+The `application-connectors` bucket contains diverse generic application connectors for the most frequently used cloud data management systems (ERP, ETL, LDAP, Reporting and Monitoring, and ServiceNow). The aim of an application connector is to facilitate and simplify the cloud application usage from your workflows.
+
```

Comment:
to facilitate and simplify the integration of a cloud application in your workflows.